provider/aws/formation/app.json.tmpl

```diff
@@ -139,6 +147,16 @@
       "Description": "The ramp up period during which a newly deployed service will receive an increasing share of traffic. Defaults to 0 seconds (disabled)",
       "Type": "String"
     },
+    "SyslogDestination": {
+      "Type": "String",
+      "Description": "Syslog address destination, you need to pass the protocol to be used, e.g. tcp+tls://logsX.syslog.com:1234",
+      "Default": ""
+    },
+    "SyslogFormat": {
+      "Type": "String",
+      "Description": "Syslog format to sent to SyslogDestination.",
+      "Default": "rfc5424"
```

Comment:
if user provide invalid `SyslogFormat` will it cause to fail?
Do we need  like allowed values validation?

Reply:
Invalid `SyslogFormat` will not update the stack and will fail. We would have to use a regex or something more complex to validate the format as it accepts ABNF reference and other RFCs.

See: 

- https://docs.docker.com/config/containers/logging/syslog/
- https://www.rfc-editor.org/rfc/rfc5424#section-6

Reply:
Sounds like it will be very hard to put a validation for something like that. I think we can proceed without that.